ena-submission/src/ena_deposition/ena_submission_helper.py

```diff
@@ -61,6 +61,8 @@ def get_ena_config(
     ena_submission_url_default: str,
     ena_reports_service_url_default: str,
 ) -> ENAConfig:
+    import dotenv
```

Comment:
why is this needed?

Reply:
You also need to set the environment variables `ENA_USERNAME` and `ENA_PASSWORD` to your ENA test account credentials. You can use the `.env` file in the root of this repository to set these variables.